components/cam/src/physics/cam/phys_grid.F90

```diff
@@ -353,6 +353,9 @@ module phys_grid
    integer, private, parameter :: def_alltoall = 1                ! default
    integer, private :: phys_alltoall = def_alltoall
 
+! Physics computational cost
+   real(r8), public :: phys_pcost ! measured physics cost on this process
```

Comment:
This is the "pre-measured" cost, right? 
Might be good to clarify that it's not currently being measured.

Reply:
No - this is the experimentally measured data. See use in physpkg.F90

Reply:
I think that's what I meant, that this is an "input" cost rather than an "output" cost

Reply:
This is an 'output" cost, if I am using your terminology correctly. It is measured during the course of the model run, and the experimentally determined value is output at the end in atm_chunk_costs.txt . It is not set by the user or the model before the run or via input.

Reply:
ah ok, I assumed it was the opposite.